vaadinfrontend/src/main/java/life/qbic/datamanager/views/projectOverview/ProjectOverviewHandler.java

```diff
@@ -32,29 +35,55 @@ public ProjectOverviewHandler(@Autowired OfferLookupService offerLookupService){
     public void handle(ProjectOverviewLayout layout) {
         if (registeredProjectOverview != layout) {
             this.registeredProjectOverview = layout;
-            configureDialogButtons();
             configureSearchDropbox();
+            configureSelectionModeDialog();
         }
     }
 
-    private void configureDialogButtons() {
-        registeredProjectOverview.searchDialog.cancel.addClickListener(e -> registeredProjectOverview.searchDialog.close());
+    private void configureSelectionModeDialog(){
+        configureSelectionModeDialogFooterButtons();
 
-        registeredProjectOverview.create.addClickListener( e-> {
-            loadItemsWithService(offerLookupService);
-            registeredProjectOverview.searchDialog.open();
-        });
+        registeredProjectOverview.selectCreationModeDialog.blankButton.addClickListener(e -> creationMode = CreationMode.BLANK);
+        registeredProjectOverview.selectCreationModeDialog.fromOfferButton.addClickListener(e -> creationMode = CreationMode.FROM_OFFER);
+    }
 
+    private void configureSelectionModeDialogFooterButtons() {
+        registeredProjectOverview.selectCreationModeDialog.next.addClickListener(e ->{
+            switch (creationMode){
+                case BLANK -> {
+                    UI.getCurrent().navigate("projects/create");
+                    registeredProjectOverview.selectCreationModeDialog.close();
+                    registeredProjectOverview.selectCreationModeDialog.reset();
+                }
+                case FROM_OFFER -> {
+                    registeredProjectOverview.selectCreationModeDialog.close();
+                    loadItemsWithService(offerLookupService);
+                    registeredProjectOverview.searchDialog.open();
+                }
+            }
+        });
+        registeredProjectOverview.selectCreationModeDialog.cancel.addClickListener(e -> {
+            registeredProjectOverview.selectCreationModeDialog.close();
+            registeredProjectOverview.selectCreationModeDialog.reset();
+        });
```

Comment:
We need to make sure the creationMode is reset if the dialogue window was closed otherwise it will remember the previous selection 👀 
```suggestion
        creationMode = CreationMode.NONE;
        });
```